server/etc/e-smith/templates/etc/dovecot/dovecot.conf/10distro-defaults

```diff
@@ -1,5 +1,6 @@
 mbox_write_locks = fcntl
+mail_shared_explicit_inbox = yes
+
 ssl_cert = </etc/pki/dovecot/certs/dovecot.pem
 ssl_key = </etc/pki/dovecot/private/dovecot.pem
-mail_shared_explicit_inbox = yes
```

Comment:
Could you revert this deletion and the addition above?

Reply:
ok, will do